### PR TITLE
Document compatibility with Django "diffsettings" command, and use with Django ready() hook

### LINKED ISSUE #49: Compatibility with diffsettings?
Hello,

I was nicely surprised to see that django apps using AppConf showed all their settings nicely in "manage.py diffsetting", imho that's a killer feature compared to all apps which hide their defaults in handmade "getattr(settings, x, default)" calls.

Maybe it'd be worth mentioning it in docs?

Would it be feasible to also make appconf inject default settings in "global_settings" of django, so that diffsettings also shows which app settings were overriden compare to their appconf defaults (with the "###")?

### PR DESCRIPTION
**Pull request by pakal**
Closes #49

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -46,7 +46,11 @@ something like the following code somewhere in your app files:
     process. Even though there are multiple modules loaded automatically,
     only the ``models`` modules (usually the ``models.py`` file of your
     app) are guaranteed to be loaded at startup. Therefore it's recommended
-    to put your ``AppConf`` subclass(es) there, too.
+    to put your ``AppConf`` subclass(es) there, too; or that you import these 
+    classes in your Django AppConfig ready_ method.
```
Comment (carltongibson) [inline README.rst]: I think I'd drop this change. I'm not sure it is right in general. `ready()` is not called until after the app registry is loaded. Often we want settings prior to that... _It depends_.

**Review by carltongibson — COMMENTED** [review]
Hi @pakal. Thanks for this. Just one comment...

**Comment by pakal** [comment]
Allright this should do then !

**Review by carltongibson — APPROVED** [review]
Super. Thanks.

**Comment by carltongibson** [comment]
Welcome aboard @pakal! ⛵️

**Pull request merged**